toolkit/docs/building/building.md

```diff
@@ -490,6 +493,16 @@ sudo make hydrate-rpms PACKAGE_ARCHIVE=./rpms.tar.gz
 > - at least one of the RPM packages mentioned in the manifest file, or
 > - the script responsible for building the chroot.
 
+#### `HYDRATED_BUILD=...`]
+
+##### `HYDRATED_BUILD=`**`y`**]
+
+> If exists, all the dependency RUN nodes will be replaced with PreBuilt Nodes if those RPMs are hydrated already. So if any dependency package fails to build, the subsequent dependent packages will not be stuck as their dependency will be satisfied by hydrated RPM. This is even applicable to the packages mentioned in REBUILD_PACKAGES. This option is introduced mainly for running PTEST.
```

Comment:
I'd only remove the last sentence, since this is public documentation and "ptest" is just a name we using in offline conversations to save time.:)